docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -23,29 +49,63 @@ SDLLockScreenConfiguration *lockScreenConfiguration = [SDLLockScreenConfiguratio
 ```swift
 let lockScreenConfiguration = SDLLockScreenConfiguration.enabled()
 ```
+!@
+
+@![android]
+`TODO Add code example`
+!@
 
 ## Customizing the Provided Lock Screen
```

Comment:
The `Showing the Device Logo` section should still be here as an android only section.

Reply:
okay